### DISCUSSION TITLE
Install from Source

### DISCUSSION THREAD

**Discussion by perrinaz** [noDRM/DeDRM_tools#359] [Q&A]
Are there instructions for a non-programmer-type to install the plugin from Source -- to get some of the updates made in the last year?

**Comment by ElleKayEm** [comment]
Click Actions near the top of the page.  Click the top workflow run.  Then click plugin at the bottom of the page.  Unzip the file that downloads and then unzip again, and you should have the 2 plugins (zip) and 3 readmes (txt) with the latest code.
